src/main/java/io/fusionauth/domain/provider/NintendoIdentityProvider.java

```diff
@@ -37,9 +37,18 @@
   @InternalJSONColumn
   public String client_secret;
 
+  @InternalJSONColumn
```

Comment:
These 3 new fields need to be added to `equals` and `hashCode`, and we can remove `SupportsPostBindings` from this class.